openid-data-classes/src/commonMain/kotlin/at/asitplus/openid/AuthorizationDetails.kt

```diff
@@ -79,6 +81,7 @@ sealed class AuthorizationDetails {
          * Credential Configuration in the [IssuerMetadata.supportedCredentialConfigurations]. The Wallet MUST use these
          * identifiers together with an Access Token in subsequent Credential Requests.
          */
+        // TODO is required in OID4VCI!
```

Comment:
Yep, see https://github.com/a-sit-plus/vck/issues/145

Reply:
For uc5 we get this after credential enrollment but this is out of scope for UC5 spec as well as CSC spec...